link-check.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "aliveStatusCodes": [200, 206],
+  "timeout": "10s",
+  "retryOn429": true,
+  "retryCount": 2,
+  "fallbackRetryDelay": "5s",
+  "ignorePatterns": [
+    {
+      "pattern": "^http://localhost:3000"
+    },
+    {
+      "pattern": "^./modules/_fuel_ts"
+    }
```

Comment:
Why do we ignore this pattern?

Reply:
@petertonysmith94 I've added this because of the [docs-api index links](https://github.com/FuelLabs/fuels-ts/pull/3430/commits/c903737ee3232cb98a6b8d4c86e212b39192348a).

Previously, these links were using relative paths. While they worked within the deployed docs-API app, accessing them directly didn’t work, and they were being incorrectly flagged as broken links (with one of them actually broken).

To fix this, I switched to using the full URL of the deployed docs-API app, which is already used in some docs guides. But I wonder if this is the best approach.

What do you think @danielbate?

Reply:
Gotcha. With the addition of #3413, referencing the full URL seems like the way forward.